docs/modeling-resources.rst

```diff
@@ -101,16 +71,79 @@ Since India does not have the equivalent of NASS CDL available to the public, we
+ 
+Russia Winter Wheat
+--------------------
+
+Since Russia does not have the equivalent of NASS CDL available to the public, we use a technique similar to the one used for our other non-US country/crop pairings. The covers were classified annually back to 2008. We did not make the covers for the entire country, but only for the select provinces and districts that are the highest producing regions.
+
+The method of crop discrimination used a thresholding of Sentinel-2 and Landsat 5/7/8 bands in a combination of Shortwave Infrared (SWIR) (~1.61 µm), Green (~0.56 µm), Near Infrared (NIR) (~0.865 µm), and Red (~0.66 µm) in the following combination: SWIR - Green/NIR + Red.
+
+A low-confidence cover was made from the yearly crop covers starting in 2008. The results of these covers can be viewed in Gro under the Land Cover (percent) metric, where the value of a pixel will represent that pixel's likelihood of growing this crop in a given year.
+ 
+Ukraine Winter Wheat
+---------------------
+
+Since Ukraine does not have the equivalent of NASS CDL available to the public, we use a technique similar to the one used for our other non-US country/crop pairings. The covers were classified annually but these annual covers only went back to 2014. This shortened time period was due to the fact that the Synthetic Aperture Radar (SAR) bands exclusive to Sentinel-1 were used for the crop discrimination step and its archival depth is not deep. 
+
+The method of crop discrimination used a thresholding of Sentinel-1 with a simple ratio between its VV band (Single co-polarization, vertical transmit/vertical receive) and the VH band (Dual-band cross-polarization, vertical transmit/horizontal receive) as follows: VV/VH.
+
+A low-confidence cover was made from the yearly crop covers starting in 2014. The results of these covers can be viewed in Gro under the Land Cover (percent) metric, where the value of a pixel will represent that pixel's likelihood of growing this crop in a given year.
+ 
+Brazil Soy
+----------
+Brazil soybeans uses a similar methodology to that of Argentina soybeans, with the exception that Landsat 5 was omitted from the constellation of satellites.
+
+Another minor change: Instead of subtracting the vegetation hue layer from the soil hue layer and isolating the top portion of the pixel values as was done for Argentina, , for Brazil we threshold and extract the soil values in each of the soil images and combine them with the other satellite soil imagery. The same is done with the vegetation values over all the vegetation images. This small technique change proved to be much more useful in finding crop areas, as it allowed for areas with poor coverage to be picked up more easily and not be drowned out.

```

Comment:
extra space and comma here `Argentina, ,`

Reply:
Good spot! Fixed.